internal/docstore/dynamodocstore/dynamo.go

```diff
@@ -12,22 +12,134 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package dynamodocstore provides a docstore implementation backed by AWS
+// DynamoDB.
+// Use OpenCollection to construct a *docstore.Collection.
+//
+// URLs
+//
+// For docstore.OpenCollection, dynamodocstore registers for the scheme
+// "dynamodocstore". The default URL opener will use an AWS session with the
```

Comment:
"dynamo" or "dynamodb" should be sufficient, here and elsewhere.

Reply:
Done.